.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,70 @@
+* Please do not leave "+1" or "me too" comments, they generate extra noise for PR followers and do not help prioritize for review
+
+
+## PR Checklist
+
+- [ ] Have you followed the guidelines in our [Contributing Documentation](../contributing/README.md)?
+- [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
+- [ ] Have you used a meaningful PR description to help maintainers and other users understand this change and help prevent duplicate work?
+Example:
+“`resource_name_here` - description of change e.g. adding property `new_property_name_here`”
+- [ ] Do your changes close any open issues? If so please include appropriate [closing keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) below.
+
+
+<!-- You can erase any parts of this template below this point that are not applicable to your Pull Request. -->
+
+
+## New Feature Submissions
+
+- [ ] Does your submission include Test coverage as described in the [Contribution Guide](../contributing/topics/guide-new-resource.md) and the tests pass? (if this is not possible for any reason, please include details of why below)
+
+
+## Changes to existing Resource / Data Source
+
+- [ ] Have you added an explanation of what your changes do and why you'd like us to include them? (This may be covered by linking to an issue above, but may benefit from additional explanation)
+- [ ] Have you written new tests for your resource or datasource changes?
+- [ ] Have you successfully run tests with your changes locally? If not, please provide details on testing challenges that prevented you running the tests.
+
+
+## Documentation Changes
+
+- [ ] Documentation is written in International English.
+- [ ] Documentation is written in a helpful and kind way to assist users that may be unfamiliar with the resource / data source.
+
+
+## Description
+
+<!-- Please include a description below with the reason for the PR, what it is doing, what it is trying to accomplish, and anything relevant for a reviewer to know. It also helps to paste the output from running the acceptance tests. -->
+
+
+
+
+## Related Issue(s)
+Fixes #0000
+## Change Log
+
+[Changelog Format](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/topics/maintainer-changelog.md)
+
+<!-- Replace the changelog example below with your entry. One resource per line. -->
+
+* `azurerm_resource` - support for the `thing1` property [GH-00000]
+
+
+<!-- What type of PR is this? -->
+
+- [ ] Bug Fix
+- [ ] New Feature
+- [ ] Enhancement
+
+
+
+
+> [!NOTE] If this PR changes meaningfully during the course of review please update the title and description as required.
```

Comment:
Should we put this other a heading so it stands out?
```suggestion
## During review and before merge:
 
 If this PR changes meaningfully during the course of review please update the title and description as required.
```

Reply:
@katbyte Somehow this lost the additional formatting I had that makes it stand out, I added it back, please see below for how it stands out. If you want a header instead of what's below, LMK. 
> [!NOTE] 
> If this PR changes meaningfully during the course of review please update the title and description as required.